app/components/public/stream/video-stream.hbs

```diff
@@ -8,5 +8,15 @@
   {{#if (eq @videoStream.videoChannel.provider 'jitsi')}}
     <Public::Stream::JitsiStream @videoStream={{@videoStream}} />
   {{/if}}
+  {{#if (and (eq @videoStream.videoChannel.provider 'youtube') this.youtubeId)}}
+    <div class="ui segment mt-4 {{if device.isMobile '' 'stream'}}">
```

Comment:
Why are you adding segment and mt-4? It doesn't make any sense?

See how BBB iframe is added!